README.asciidoc

```diff
@@ -15,39 +15,34 @@ Elasticsearch is a distributed RESTful search engine built for the cloud. Featur
 ** Index level configuration (number of shards, index storage, ...).
 * Various set of APIs
 ** HTTP RESTful API
-** Native Java API.
 ** All APIs perform automatic node operation rerouting.
 * Document oriented
 ** No need for upfront schema definition.
 ** Schema can be defined for customization of the indexing process.
 * Reliable, Asynchronous Write Behind for long term persistency.
 * (Near) Real Time Search.
-* Built on top of Lucene
+* Built on top of Apache Lucene
 ** Each shard is a fully functional Lucene index
 ** All the power of Lucene easily exposed through simple configuration / plugins.
 * Per operation consistency
 ** Single document level operations are atomic, consistent, isolated and durable.
 
-h2. Getting Started
+== Getting Started
 
 First of all, DON'T PANIC. It will take 5 minutes to get the gist of what Elasticsearch is all about.
 
-h3. Requirements
+=== Installation
 
-You need to have a recent version of Java installed. See the "Setup":http://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html#jvm-version page for more information.
-
-h3. Installation
-
-* "Download":https://www.elastic.co/downloads/elasticsearch and unzip the Elasticsearch official distribution.
-* Run @bin/elasticsearch@ on unix, or @bin\elasticsearch.bat@ on windows.
-* Run @curl -X GET http://localhost:9200/@.
+* https://www.elastic.co/downloads/elasticsearch[Download] and unpack the Elasticsearch official distribution.
+* Run `bin/elasticsearch` on unix, or `bin\elasticsearch.bat` on windows.
+* Run `curl -X GET http://localhost:9200/`.
 * Start more servers ...
 
-h3. Indexing
+=== Indexing
 
-Let's try and index some twitter like information. First, let's index some tweets (the @twitter@ index will be created automatically):
+Let's try and index some twitter like information. First, let's index some tweets (the `twitter` index will be created automatically):
 
-<pre>
+-------------------------------------
```

Comment:
Code blocks only [need](https://asciidoctor.org/docs/user-manual/#built-in-blocks-summary) four dashes `----`. I think it'd be nice to go with that while we're here. No big deal, but it'd be nice.

Reply:
done